crates/bevy_xpbd_2d/examples/one_way_platform_2d.rs

```diff
@@ -0,0 +1,305 @@
+        .run();
+}
+
+#[derive(Component)]
+struct Actor;
+
+#[derive(Clone, Eq, PartialEq, Debug, Default, Component)]
+pub struct OneWayPlatform(HashSet<Entity>);
+
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Default, Component, Reflect)]
+pub enum PassThroughOneWayPlatform {
+    #[default]
+    /// Passes through a `OneWayPlatform` if the contact normal is in line with the platform's local-space up vector
+    ByNormal,
+    /// Always passes through a `OneWayPlatform`, temporarily set this to allow an actor to jump down through a platform
+    Always,
+    /// Never passes through a `OneWayPlatform`
+    Never,
+}
+
+fn setup(
+    mut commands: Commands,
+    mut materials: ResMut<Assets<ColorMaterial>>,
+    mut meshes: ResMut<Assets<Mesh>>,
+) {
+    commands.spawn(Camera2dBundle::default());
+
+    // For borders
+    let square_sprite = Sprite {
+        color: Color::rgb(0.7, 0.7, 0.8),
+        custom_size: Some(Vec2::splat(50.0)),
+        ..default()
+    };
+
+    // Ceiling
+    commands.spawn((
+        SpriteBundle {
+            sprite: square_sprite.clone(),
+            transform: Transform::from_scale(Vec3::new(20.0, 1.0, 1.0)),
+            ..default()
+        },
+        RigidBody::Static,
+        Position(Vector::Y * 50.0 * 6.0),
+        Collider::cuboid(50.0 * 20.0, 50.0),
+    ));
+    // Floor
+    commands.spawn((
+        SpriteBundle {
+            sprite: square_sprite.clone(),
+            transform: Transform::from_scale(Vec3::new(20.0, 1.0, 1.0)),
+            ..default()
+        },
+        RigidBody::Static,
+        Position(Vector::NEG_Y * 50.0 * 6.0),
+        Collider::cuboid(50.0 * 20.0, 50.0),
+    ));
+    // Left wall
+    commands.spawn((
+        SpriteBundle {
+            sprite: square_sprite.clone(),
+            transform: Transform::from_scale(Vec3::new(1.0, 11.0, 1.0)),
+            ..default()
+        },
+        RigidBody::Static,
+        Position(Vector::NEG_X * 50.0 * 9.5),
+        Collider::cuboid(50.0, 50.0 * 11.0),
+    ));
+    // Right wall
+    commands.spawn((
+        SpriteBundle {
+            sprite: square_sprite,
+            transform: Transform::from_scale(Vec3::new(1.0, 11.0, 1.0)),
+            ..default()
+        },
+        RigidBody::Static,
+        Position(Vector::X * 50.0 * 9.5),
+        Collider::cuboid(50.0, 50.0 * 11.0),
+    ));
+
+    // For one-way platforms
+    let one_way_sprite = Sprite {
+        color: Color::rgba(0.7, 0.7, 0.8, 0.25),
+        custom_size: Some(Vec2::splat(50.0)),
+        ..default()
+    };
+
+    // Spawn some one way platforms
+    for y in -2..=2 {
+        commands.spawn((
+            SpriteBundle {
+                sprite: one_way_sprite.clone(),
+                transform: Transform::from_scale(Vec3::new(10.0, 0.5, 1.0)),
+                ..default()
+            },
+            RigidBody::Static,
+            Position(Vector::Y * 15.0 * 6.0 * y as Scalar),
```

Comment:
The distance between the ground and the first platform seems to be 15px larger than the distance between the platforms. We should probably either move the ground up or the platforms down.

```suggestion
            Position(Vector::Y * 15.0 * 6.0 * y as Scalar - Vector::Y * 15.0),
```

Reply:
I was going to disagree with you on this because I preferred the way it looked wrt the borders at 15 spacing, but after changing jump strength to 450 (which I did agree with) we can no longer reach the bottom platform from the ground(!)

I'm not 100% sure myself, but this does _appear_ to be the correct height so that the spacing is even, so let's go with this.